web-api/src/persistence/postgres/caseCorrespondences/schema.ts

```diff
@@ -1,13 +1,13 @@
 import { CaseTable } from '@web-api/persistence/postgres/cases/schema';
-import { Selectable, Insertable, Updateable } from 'kysely';
+import { Selectable, Insertable } from 'kysely';
 
 const DEFAULT = {};
 
 export const caseCorrespondenceTableDefinition = {
-  archived: DEFAULT as boolean | undefined,
+  archived: DEFAULT as boolean | null,
```

Comment:
`undefined` is not a Postgres type. While there are Kysely-specific reasons we could use `undefined`, I do not believe we are leveraging them, and I think it is probably better to use `null`.